src/Material/Button.elm

```diff
@@ -263,18 +263,18 @@ view lift model config html =
       [ cs "mdl-button"
       , cs "mdl-js-button"
       , cs "mdl-js-ripple-effect" `when` summary.config.ripple
+      , listeners
       ]
-      (List.concat [startListeners, stopListeners, misc, type']
+      (List.concat [stopListeners, misc, type']
          |> List.filterMap identity)
       (if summary.config.ripple then
           List.concat
             [ html
             -- Ripple element must be last or blurAndForward hack fails.
             , [ Html.App.map lift <| Ripple.view'
                   [ class "mdl-button__ripple-container"
-                  --, Helpers.blurOn "mouseup"
-                  , Ripple.upOn "blur"
-                  , Ripple.upOn "touchcancel"
+                  -- , Ripple.upOn "blur"
```

Comment:
It did not but I am unable to have it break with just having the button events trigger `Ripple.upOn`